src/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 set(SOURCES
-        Guid.cpp)
+        guid.cpp)
```

Comment:
What happened here?

Reply:
Reminder from discussion earlier: Case-sensitivity in Unix-based OSes causes CMake to fail so I just had it match the source file naming.